public/stylesheets/classes.css

```diff
@@ -27,6 +37,10 @@
   border: none;
 }
 
+p {
```

Comment:
- Try not to use html tag selectors when styling a specific page as it affects the same element in all pages